frontend/src/scenes/pipeline/AppsManagement.tsx

```diff
@@ -0,0 +1,394 @@
+                                    title="Are you sure you wish to uninstall this app completely?"
+                                    onConfirm={() => uninstallPlugin(plugin.id)}
+                                    okText="Uninstall"
+                                    cancelText="Cancel"
+                                    className="Plugins__Popconfirm"
+                                >
+                                    <LemonButton
+                                        type="primary"
+                                        status="danger"
+                                        size="small"
+                                        icon={<IconDelete />}
+                                        disabledReason={
+                                            unusedPlugins.includes(plugin.id) ? undefined : 'This app is still in use.'
+                                        }
+                                        data-attr="plugin-uninstall"
+                                    >
+                                        Uninstall
+                                    </LemonButton>
+                                </Popconfirm>
+                            )
+                        },
+                    },
+                ]}
+            />
+        </>
+    )
+}
+
+function OutOfSyncApps(): JSX.Element {
+    const { missingGlobalPlugins, shouldBeGlobalPlugins, shouldNotBeGlobalPlugins } = useValues(appsManagementLogic)
+    if (
+        missingGlobalPlugins.length === 0 &&
+        shouldBeGlobalPlugins.length === 0 &&
+        shouldNotBeGlobalPlugins.length === 0
+    ) {
+        return <></>
+    }
+    return (
+        <>
+            <LemonBanner type="warning"> Apps state is out of sync </LemonBanner>
+            <br />
+            <MissingGlobalPlugins />
+
+            {shouldNotBeGlobalPlugins && (
+                <>
+                    <LemonLabel> Apps that should NOT be global </LemonLabel>
+                    <p>These apps should NOT be global according to repo.</p>
+                    <AppsTable plugins={shouldNotBeGlobalPlugins} />
+                </>
+            )}
+
+            {shouldBeGlobalPlugins && (
+                <>
+                    <LemonLabel> Apps that should be global </LemonLabel>
+                    <p>These already installed apps should be global according to repo.</p>
+                    <AppsTable plugins={shouldBeGlobalPlugins} />
+                </>
+            )}
+            <LemonDivider />
+        </>
+    )
+}
+
+function MissingGlobalPlugins(): JSX.Element {
+    const { missingGlobalPlugins } = useValues(appsManagementLogic)
+    const { installPlugin } = useActions(appsManagementLogic)
+
+    if (missingGlobalPlugins.length === 0) {
+        return <></>
+    }
+    const data = missingGlobalPlugins.map((url: string) => ({ url }))
+    return (
+        <>
+            <LemonLabel> Missing Global Plugins </LemonLabel>
```

Comment:
Some of these headings are Title Case. We should follow our brand style by sticking to Sentence case everywhere.